test/e2e/basics/render.test.js

```diff
@@ -75,4 +75,25 @@ test.describe("Render MapML resources test", () => {
         const map = await page.$("xpath=//html/body/mapml-viewer");
         await expect(map).not.toEqual(null);
     });
+
+    test("Projection defaults to OSMTILE in the case of unknown projection", async () => {
+        //Changes page.goto response (initial page load) to be of content type text/mapml
+        await page.route("test/e2e/basics/test.mapml", async route => {
+            const response = await page.request.fetch("test/e2e/basics/test.mapml");
```

Comment:
Should be unknown_projection.mapml instead of test.mapml as seen in the previous commit.